src/Redux/actionCreators/RequestActions.js

```diff
@@ -112,15 +143,45 @@ const updateRequestSummary = data => dispatch => {
   dispatch({ type: UPDATE_REQUEST_SUMMARY, payload: data });
 };
 
+const fetchRequestSummaryError = () => dispatch => {
+  // On Error set the summary data to default Zeros
+  var data = {
+    PENDING: 0,
+    ACTIVE: 0,
+    SOLUTION_VOTE: 0,
+    COMPLETED: 0,
+    INCOMPLETE: 0,
+    REJECTED: 0,
+    CLOSED: 0,
+  };
+  dispatch({ type: UPDATE_REQUEST_SUMMARY, payload: data });
+};
+
 // Fetching The the Request Solution
 const fetchRequestSolutionAPI = async requestId => {
   const url = `${APIEndpoints.RFAI.endpoint}/request/${requestId}${APIPaths.RFAI_REQUEST_SOLUTION}`;
-  const response = await fetch(url);
-  return response.json();
+  // const response = await fetch(url);
```

Comment:
comments could be removed

Reply:
Will remove and will be available in the next PR.